### DISCUSSION TITLE
Which the difference of setup a emqx cluster using emqx-operator and use emqx helm chart

### DISCUSSION THREAD

**Discussion by nielsonsantana** [emqx/emqx-operator#738] [Q&A] [answered]
Which the difference of setup a emqx cluster using emqx-operator and use emqx helm chart

**Comment by jacky-xbb** [comment]
The most significant difference is customization. Emqx operator is highly customizable via CR, and emqx helm is limited to customization. 
Emqx operator has integrated some important features, e.g. load rebalancing(will be released), blue-green upgrade, and emqx plugins management.

**Comment by jacky-xbb** [comment, marked as answer]
Main features of EMQX Operator:
- Simplified Deployment: EMQX Operator provides a Kubernetes-native way to deploy EMQX on a Kubernetes cluster, simplifying the deployment process.

- Automatic Scaling: You can expand the EMQX  cluster without interrupting your business to access more devices and carry more complex business.

- Load Rebalancing(will be released): After the cluster has been scaled, the connection of the old nodes will be smoothly migrated to the new nodes, thus achieving a more balanced distribution of the entire cluster load.

- On-Fly Updating: EMQX Operator allows users to perform updates on EMQX configuration and ensures that the application remains available during the update process.

- EMQX Upgrading: Perform blue-green upgrades EMQX without affecting business operations.

- EMQX Monitoring: Monitor all metrics of the entire EMQX cluster through Emqx Exporter, Prometheus, and Grafana.

**Reply by nielsonsantana** [reply]
Hi @jacky-xbb, thanks for you explanation.
